wellcomecollection/wellcomecollection.org
# Toggles and Error page changes

## What does this change?

It's happened a few times that we were flagged error pages that were caused by toggles (usually `Staging API` which I believe gets confused with `API Toolbar`). As API Toolbar is the main one that's used by most people, especially outside the team, it makes sense to move it to the top and try to avoid confusion. I've split the toggles a bit differently and added another level of headings.
Also adds possibility for linking to documentation (see first Digital team permanent toggle).

<img width="1226" alt="Screenshot 2024-08-07 at 10 58 37" src="https://github.com/user-attachments/assets/cc6dc068-5cb4-4159-9904-329b41fd7c48">




I've also modified the Error page "toggle" message to try and make it clearer it might be a problem. I've change the message and moved the block to the top of the page, as well as added a function that fetched the readable name based off of my chat with @pollecuttn below.

<img width="1011" alt="Screenshot 2024-08-07 at 12 35 17" src="https://github.com/user-attachments/assets/1f6accd2-8b05-4c38-9153-a15f16a02a62">
<img width="1021" alt="Screenshot 2024-08-07 at 12 35 05" src="https://github.com/user-attachments/assets/705e59bc-eab4-41d5-a908-ae085a61b7a3">



## How to test

Run dash locally and go to a 404 page with at least one toggle set to TRUE to see the toggles message

## How can we measure success?

Users gain autonomy in debugging themselves and are less likely to activate the wrong toggle 

## Have we considered potential risks?

Shouldn't be any, I'm not changing the functionalities.
